iree/test/e2e/xla_ops/BUILD

```diff
@@ -280,6 +283,7 @@ iree_check_single_backend_test_suite(
         include = ["*.mlir"],
         exclude = [
             "fft.mlir",  # TODO(GH-6388): Enable the test.
+            "rng.mlir",
```

Comment:
I'm surprised that it's not working on SPIR-V side. What's the error? Could you file an issue for this?

Reply:
I don't have spirv driver installed haha 😆, I'll install and try

Reply:
You can also push the commit, and the bot will run it.